build-system/compile/sources.js

```diff
@@ -30,7 +30,6 @@ const COMMON_GLOBS = [
   'third_party/inputmask/**/*.js',
   'third_party/mustache/**/*.js',
   'third_party/react-dates/bundle.js',
-  'third_party/optimized-svg-icons/social-share-svgs.js',
   'third_party/set-dom/set-dom.js',
```

Comment:
@dvoytenko Is there someone you would recommend to review this file?  I changed it to pass Travis, but am not very familiar with this part of the codebase.

Reply:
No. My review should be enough.

Reply:
Sounds good!